### ISSUE TITLE
Bleachbit 4.5.0.2499 - After a recent Window update, Bleachbit now produces a surprising error due to Edge automatically running.

### ISSUE THREAD

**Issue by m0lz** [bleachbit/bleachbit#1556]
BleachBit 4.5.0.2499 in use.

When run I have it set to clean out a few things from Microsoft Edge
Cache/Cookies/Dom Storage/Form History/History

I generally do not use Edge, I have Brave installed as my Default Browser.

I recently had an update to Windows ..
2023-09 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5030310)

Which I believe has introduced something new, by default it now adds Microsoft Edge into the Startup Apps ..

![Untitled](https://github.com/bleachbit/bleachbit/assets/20239221/d641b8fd-1104-4cda-be75-1d26e1f988ed)

Problem is:
When I run Bleachbit it throws an error, cant clean MS Edge due to it running.

This came as a bit of a shock, I do not use Edge. And I think this is going to be problematic for Bleachbit, if Windows update is automatically adding it to startup apps for everyone, even when Edge is not the default Browser as chosen by the user.

I have never had Edge starting up with windows before, and since I last run Bleachbit about six days ago, the only thing that could have altered the startups before today would have been the Windows update as mentioned above.

---------------

A bit of speculation I dont really think is relevant to this issue, but maybe worth mentioning :
Microsoft have also added a new AI App to Windows 11 called Copilot (I have disabled it as its something I do not wish to use), and I wonder if that for some reason has made Microsoft deem Edge to be running as a necessary default, even if Edge is not the default Browser. Maybe not relevant, and certainly confusing as to why Microsoft would think someone with a Default Browser already set would need another browser running in the background. Annoying clowns.

I cant see how you can solve this as it needs the user to disable Edge starting up automatically, but thought you need to be made aware of the issue.

Kind regards, m0lz

**Comment by dmitriy1980** [comment]
Open Edge and make sure these 2 options are off. They are the reason Edge is running in the background. Deselect these and try running again and see if that works.
![image](https://github.com/bleachbit/bleachbit/assets/69124875/8114aeee-7d0e-4b3e-ba1d-0c44b13bcd63)

**Comment by az0** [comment]
@m0lz 
Did the suggestion from dmitriy1980 help?

**Comment by XhmikosR** [comment]
Yeah, that's the proper way to stop Chromium-based browsers from running in the background

**Comment by m0lz** [comment]
Those two settings in Edge settings as pointed out by dmitriy1980 were already set to disabled (months ago).

The new thing that happened was an update to windows which added Edge once again to the Startup Apps (or maybe it could have been an update to Edge along with a Windows update .. idk for sure).

The main point of my post is I had not personally allowed Edge to be a Startup App, so whatever update did this did it without my say so, and so caused the surprise of Edge running in the background when it should not have been.
